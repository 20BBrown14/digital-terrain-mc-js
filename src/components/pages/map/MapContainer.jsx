import React from 'react';
import MapView from './MapView';

/**
 * Map page container.
 */
const MapContainer = () => (
  <MapView />
);

export default MapContainer;
