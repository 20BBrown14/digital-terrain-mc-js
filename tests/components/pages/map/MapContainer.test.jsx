import React from 'react';
import MapContainer from '../../../../src/components/pages/map/MapContainer';

describe('MapContainer', () => {
  it('renders a default view', () => {
    const container = render(<MapContainer />);
    expect(container).toMatchSnapshot();
  });

  it('has a MapView', () => {
    const container = shallow(<MapContainer />);
    expect(container.find('MapView')).toHaveLength(1);
  });
});
