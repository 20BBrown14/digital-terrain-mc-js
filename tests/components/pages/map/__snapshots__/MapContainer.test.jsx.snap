// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MapContainer renders a default view 1`] = `
<div>
  <iframe
    class="map-iframe"
    src="https://dynmap-reverse-proxy.herokuapp.com/"
    title="Digital Terrain Dynmap"
  >
    &lt;p&gt;Your browser does not support this type of content.&lt;/p&gt;
  </iframe>
</div>
`;
